### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by AntiPontifex**
Still updating

### CODE REVIEW

File: streams/cn_streamlink.m3u [hunk 1]
```diff
@@ -51,43 +51,43 @@ https://sc2022.stream-link.org/tv2306.php?id=e08
...
 https://sc2022.stream-link.org/tv2306.php?id=e12
 #EXTINF:-1 tvg-id="",Now新聞台
 https://sc2022.stream-link.org/tv2306.php?id=h08
 #EXTINF:-1 tvg-id="",Now精選體育
 https://sc2022.stream-link.org/tv2306.php?id=s01
 #EXTINF:-1 tvg-id="",PLAYY 韓國電影台
 https://sc2022.stream-link.org/tv2306.php?id=e19
-#EXTINF:-1 tvg-id="",SKY News
+#EXTINF:-1 tvg-id="SkyNews.uk",SKY News
 https://sc2022.stream-link.org/tv2306.php?id=e13
```
Comment (freearhey) [inline streams/cn_streamlink.m3u:89]:
```suggestion
```

File: streams/cn_streamlink.m3u [hunk 2]
```diff
@@ -37,11 +37,11 @@ https://sc2022.stream-link.org/tv2306.php?id=e03
...
 #EXTINF:-1 tvg-id="",CBS News
 https://sc2022.stream-link.org/tv2306.php?id=e05
-#EXTINF:-1 tvg-id="",CNA News
+#EXTINF:-1 tvg-id="CNAInternational.sg",CNA News
 https://sc2022.stream-link.org/tv2306.php?id=e06
 #EXTINF:-1 tvg-id="",CNN Live
 https://sc2022.stream-link.org/tv2306.php?id=e07
-#EXTINF:-1 tvg-id="",DAZN F1
+#EXTINF:-1 tvg-id="DAZNF1.es",DAZN F1
 https://sc2022.stream-link.org/tv2306.php?id=s07
```
Comment (freearhey) [inline streams/cn_streamlink.m3u:45]:
```suggestion
```

**Review by freearhey — CHANGES_REQUESTED** [review]
To make the error disappear you need to remove these two channels from the playlist.

```sh
/cn_streamlink.m3u
 44    error    "DAZN F1" is on the blocklist due to claims of copyright holders (https://github.com/github/dmca/blob/master/2020/09/2020-09-16-dfl.md)
 88    error    "SKY News" is on the blocklist due to claims of copyright holders (https://github.com/iptv-org/iptv/issues/7[31](https://github.com/iptv-org/iptv/actions/runs/7020426177/job/19100103340?pr=15175#step:6:32)4)
```

https://github.com/iptv-org/iptv/actions/runs/7020426177/job/19100103340?pr=15175

**Review by Dum4G — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Comment by AntiPontifex** [comment]
@freearhey Forgot to add: thank you for your help

**Pull request merged**